frontend/src/routes/Governance/policies/policy-details/PolicyTemplateDetailsPage.tsx

```diff
@@ -40,7 +42,15 @@ export function PolicyTemplateDetailsPage() {
     <AcmPage
       header={
         <AcmPageHeader
-          title={templateName}
+          title={
+            <>
+              <TemplateDetailTitle
+                policyKind={template?.kind}
```

Comment:
You already have the kind from above, could you specify it directly rather than wait for the search query to return? This would make the loading less jarring since the policy name wouldn't shift over to the right after it loads.